### PR TITLE
FE: Initial Services Tab on Node Status, with initial mocked Windows Services table

### PR DESCRIPTION
**Pull request by Shahbaz-dataq**
## Description
On the Node Status page, create a new Services table which will display service inventory data from Windows Management and Service Discovery/Monitoring. However, we don’t have the actual data yet, so we will just create a UI with mock data.

Tasks:

Create a Services tab.

Create an initial Windows Service Summary section with a table as shown below (inside the red rectangle). Add Search, Download CSV and Refresh icons, but they don’t have any functionality

Create 3 subtabs with static text as below. For now don’t display the numbers like (3/10)

Display data as below, it is just mock data from now. We will work on getting actual data in subsequent issues

Add Pagination component but doesn’t need to work yet

We’ll add Hosted Services in a subsequent issue

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2558

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Review by Syed-Bilal-Haider-Dataq — COMMENTED** [review]
Looks good

File: ui/src/components/NodeStatus/NodeServices.vue [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="window-service">
+      <TableCard>
+          <div>
+              <section class="feather-row">
+                  <div class="heading-container">
+                    <h3 data-test="heading" class="feather-col-6">Windows Service Summary</h3>
+                    <h5>10 Services Discoveres</h5>
```
Comment (synqotik) [inline ui/src/components/NodeStatus/NodeServices.vue:8]: Should be `Discovered`

File: ui/src/components/NodeStatus/ServiceTable.vue [hunk 2]
```diff
@@ -0,0 +1,244 @@
+<template>
+    <div class="container">
+      <table class="data-table" aria-label="Window Services Table">
```
Comment (synqotik) [inline ui/src/components/NodeStatus/ServiceTable.vue:3]: `Window` -> `Windows`

File: ui/src/components/NodeStatus/ServiceTable.vue [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+import AddNote from '@featherds/icon/action/AddNote'
+import { SORT } from '@featherds/table'
+import { cloneDeep, sortBy } from 'lodash'
+import { ServiceType } from '../utils';
+
+const emptyListContent = {
+  msg: 'No results found.'
+}
+
+const dummyObjests = [
```
Comment (synqotik) [inline ui/src/components/NodeStatus/ServiceTable.vue:79]: Should be `dummyObjects`

File: ui/src/components/NodeStatus/ServiceTable.vue [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+      />
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { Pagination } from '@/types/alerts'
+import AddNote from '@featherds/icon/action/AddNote'
+import { SORT } from '@featherds/table'
+import { cloneDeep, sortBy } from 'lodash'
+import { ServiceType } from '../utils';
```
Comment (synqotik) [inline ui/src/components/NodeStatus/ServiceTable.vue:73]: No semicolon

**Review by synqotik — APPROVED** [review]
Looks good! Just a couple small typos to fix before merging.

**Pull request merged**